app/views/subscriber/group/_roles.html.haml

```diff
@@ -5,9 +5,9 @@
 
 - if @role_types
   - @role_types.each do |layer, groups|
-    %h4.filter-toggle=layer
+    %h3.filter-toggle.mt-4=layer if groups.any?
```

Comment:
In welchem Fall ist dieses `if` nützlich? Man muss doch immer zuerst mit dem Typeahead eine Gruppe auswählen, und erst dann wird dieser Abschnitt überhaupt angezeigt (via `if @role_types` zwei Zeilen weiter oben?)
Präsenz von `@role_types` impliziert meiner Ansicht nach `groups.any?`, sehe ich das falsch?

Reply:
Ja, es wird immer zuerst eine Gruppe ausgewählt und dann entsprechend die @role_types befüllt. 

Falls jetzt der Layer groupen mit Rollen enthält (weil dieses als global betrachtet werden wie zb beim Sac) dann erscheinen aktuell nur die Layer als überschriften aber ohne auswählbare Rollen, siehe screenshot 

![image](https://github.com/hitobito/hitobito/assets/180780/e128f0d7-4aa2-4a77-a717-a687cf702a0f)

Habe die Anpassung auch noch im `role` partial beim `people_filter` nachgezogen